src/server/hset_family.cc

```diff
@@ -408,10 +408,14 @@ OpResult<vector<OptStr>> OpHMGet(const OpArgs& op_args, std::string_view key, Cm
   if (pv.Encoding() == kEncodingListPack) {
     uint8_t* lp = (uint8_t*)pv.RObjPtr();
 
-    absl::flat_hash_map<string_view, unsigned> reverse;
+    absl::flat_hash_map<string_view, std::vector<size_t>> reverse;
```

Comment:
it's gonna be more cpu intensive because for 99.999% of the usecases vector is not needed.
Can you please use InlinedVector instead?